connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoWriter.scala

```diff
@@ -237,7 +247,14 @@ object KustoWriter {
         val blobPath = blobUri + sas
         val blobSourceInfo = new BlobSourceInfo(blobPath, size)
 
-        partitionsResults.add(PartitionResult(ingestClient.ingestFromBlob(blobSourceInfo, props), partitionId))
+        val res: IngestionResult = try {
+          KDSU.logInfo(myName, s"Queued blob for ingestion in partition ${TaskContext.getPartitionId} for requestId: '$requestId}")
+          ingestClient.ingestFromBlob(blobSourceInfo, props)
+        } catch {
+          case _: IngestionClientException =>
+            ingestClient.ingestFromBlob(blobSourceInfo, props)
```

Comment:
Is this a retry?
Why this particular set exceptions?

Reply:
will add a comment